webapp/src/client.js

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable no-useless-catch */
+import axios from 'axios';
+
+class Client {
+    constructor() {
+        this.axiosInstance = axios.create({
+            baseUrl: '/plugins/com.mattermost.digitalocean',
+        });
+    }
+
+    getDOTeamRegions = async () => {
+        return this.doGet('/api/v1/get-do-regions');
+    }
+
+    getDOTeamDropletSizes = async () => {
```

Comment:
Do you know that sizes depend on the region? Some sizes are only available in specific regions. The Regions endpoint provides a list of sizes available for each region. Sample response for get regions

```
{
   "regions":[
      {
         "name":"New York 1",
         "slug":"nyc1",
         "sizes":[

         ],
         "features":[
            "virtio",
            "backups"
         ],
         "available":false
      },
      {
         "name":"Amsterdam 1",
         "slug":"ams1",
         "sizes":[

         ],
         "features":[
            "virtio",
            "backups"
         ],
         "available":false
      },
      {
         "name":"San Francisco 1",
         "slug":"sfo1",
         "sizes":[
            "s-1vcpu-1gb",
            "s-1vcpu-2gb",
            "s-1vcpu-3gb",
            "s-2vcpu-2gb",
            "s-3vcpu-1gb",
            "s-2vcpu-4gb",
            "s-4vcpu-8gb",
            "s-6vcpu-16gb",
            "s-8vcpu-32gb",
            "s-12vcpu-48gb",
            "s-16vcpu-64gb",
            "s-20vcpu-96gb",
            "s-24vcpu-128gb",
            "s-32vcpu-192gb"
         ],
         "features":[
            "virtio",
            "backups",
            "metadata"
         ],
         "available":true
      }
   ]
}
```

Reply:
Let me work on this

Reply:
@phillipahereza how representative is this sample response? Does that mean New York 1 has no sizes?

Reply:
It's a dummy response, I was just showing you the structure of the response